### ISSUE TITLE
Cannot copy sent message if it contains an IP address - tapping just not popup menu

### ISSUE THREAD

**Issue by ghsrcgh** [redsolution/xabber-android#749]
Cannot copy sent message if it contains an IP address - tapping just not popup menu with the options (to copy or any other). Other messages (received, sent) are tapped and copied without any issues.

Sample message that was sent (without quotes):
"95.221.150.18 test this ip if possible ... and other text here"

When you try to tap this message - nothing appears. IP address is underlined.

OS: Android 7.0
Samsung Galaxy S7

**Comment by casperklein** [comment]
Works for me:

![Screenshot](http://i.imgur.com/xqZk79h.jpg)

If I single tab right next to the received text (red circle), the popup menu appears.

OS: Android 7.0
Samsung Galaxy S7
Xabber: 2.0.2(369)

**Comment by ghsrcgh** [comment]
Agree, it will work if you will tap righter than received message, or lefter (near) than sent message. In that cases menu will popup and copy available.

But if you will tap on the message text (with that text) - menu will not popup. I'm sure this is a bug.
